Resources/Prototypes/Maps/rim.yml

```diff
@@ -0,0 +1,63 @@
+- type: gameMap
+  id: rim
+  mapName: 'rim'
+  mapPath: /Maps/rim.yml
+  minPlayers: 0
+  maxPlayers: 60
+  fallback: true
```

Comment:
for what is that?

Reply:
You are going to need to be more specific

Reply:
why is this line here? for what purpose? what is it doing?

Reply:
What line?

Reply:
`fallback: true`